src/layer.rs

```diff
@@ -349,19 +349,14 @@ impl Layer {
 
         self.layerinfo_to_file_if_needed(path, opts)?;
 
-        for (name, glyph_path) in self.contents.iter() {
```

Comment:
It looks like this is duplicate code for the glif file writes and should be removed elsewhere if this PR is not merged

Reply:
> I thiiiink what I'd like to do is revert that previous commit, and then I'll redo it and redo my measurements. I'll do that (the reverting) and then you'll need to patch up this PR.

Np, will do